### DISCUSSION TITLE
xstate inspect

### DISCUSSION THREAD

**Discussion by khairold** [statelyai/xstate#2166] [Q&A]
Hi,

Can anyone please tell me why the code below works well on https://xstate.js.org/viz/, but not with @xstate/inspect ?

```
const addressMachine = Machine({
  id: 'address',
  initial: 'found',
  states: {
    found: { type: 'final' },
  },
})
```

**Comment by Andarist** [comment]
Please always try to share a repro case in a runnable form - either by providing a git repository to clone or a codesandbox. OSS maintainers usually can't afford the time to set up the repro, even if exact steps are given.
